### DISCUSSION TITLE
Crash back to the login page

### DISCUSSION THREAD

**Discussion by SuperBen118** [danny-avila/LibreChat#3495] [Troubleshooting]
### What happened?

Using webservers for deployment, after logging in, if there is no operation for a few minutes during the use, then inputting will cause the application to crash back to the login interface, and this happens on both AWS and AZURE. However, if deployed on platforms like Sealos, similar issues do not occur. What could be the reason, and how can this be resolved?

### Steps to Reproduce

1.deploy on webservers, like AWS
2.sign up and log in
3.do whatver and pause for a few mininutes
4.continue inputting and see the application crash back to the login page

### What browsers are you seeing the problem on?

Microsoft Edge

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
There are several discussions on this topic and it's not a bug but a native web feature concerning HTTPS:

https://github.com/danny-avila/LibreChat/discussions/3001
